addons/account_fleet/views/fleet_vehicle_cost_views.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id='fleet_vehicle_log_services_view_form' model='ir.ui.view'>
+        <field name="name">fleet.vehicle.log.services.form.inherit.account</field>
+        <field name="model">fleet.vehicle.log.services</field>
+        <field name="inherit_id" ref="fleet.fleet_vehicle_log_services_view_form"/>
+        <field name="priority" eval="1" />
+        <field name="arch" type="xml">
+            <xpath expr="//sheet" position="inside">
+                <field name="account_move" invisible="1"/>
+                <field name="account_move_line" invisible="1"/>
+                <field name="account_move_state" invisible="1"/>
```

Comment:
I think it's not necessary anymore since odoo/odoo#137031. If it needs to be kept, a comment needs to be added behind to explain why (ensure saving of the field,...) :)